docs/readthedocs/usersguide.md

```diff
@@ -1,85 +1,82 @@
+You can either delegate the job to your preferred IDE or build an executable jar manually and use a command line interface (CLI).
+
+### Using a CLI
 
-#. 
-  Execute ``./gradlew shadowJar`` within the CLI. This creates a fat jar of SIMONA inside of the directory ``build/libs``.
+In order to be able to execute SIMONA from a CLI, you need an executable fat jar. A fat jar contains all compiled classes and dependencies, ready for execution. For building one, you can use a Gradle task of the project.
+
+#.
+Open a CLI and change directories to the top level directory of the project.
+
+#.
+Execute ``./gradlew shadowJar`` within the CLI. This creates a fat jar of SIMONA inside of the directory ``build/libs``.
 
 #.
-  For executing a fat jar you need to specify the classpath of the entrypoint of your application. Assuming we are still in the top level directory of our project, the execution command would look as follows:
+For executing a fat jar you need to specify the classpath of the entrypoint of your application. Assuming we are still in the top level directory of our project, the execution command would look as follows:
 
-  .. code-block:: none
+.. code-block:: none
 
       java -cp build/libs/simona-2.1-all.jar edu.ie3.simona.main.RunSimonaStandalone
 
 
-Using an IDE  
-------------
+### Using an IDE
+
 If you use an integrated development environment (IDE) all you have to do is navigate to the class, and press ``right-click -> run``. The IDE will compile and then execute the code for you.
 
 When executed, the simulation starts at a specified start time and simulates the system interactions until the specified end time.
 As you might wonder who tells SIMONA when to start, when to end and what to actually perform a simulation on, there is indeed a piece missing.
 Since a simulation can be run with all sorts of grids and with different parameter sets, a simulation configuration has to be set for successful execution.
 
-Simulation Inputs
-=================
+## Simulation Inputs
+
+
+### Default Config and Where to Find It
 
-Default Config and Where to Find It
------------------------------------
 
 Within the SIMONA repository, there are two default configurations and their corresponding grids with all necessary data to execute a simulation.
 For each configuration, there is a separate directory in the project directory at ``./inputData``.
 Each configuration is described by a ``.conf`` file that holds the simulation configuration parameters.
 
 
-Set the Simulation Configuration
---------------------------------
-When running a simulation, SIMONA expects the directory of the configuration file as a command-line argument. 
+### Set the Simulation Configuration
+
+When running a simulation, SIMONA expects the directory of the configuration file as a command-line argument.
 Let's pretend you want to start a simulation with the default ``vn_simona`` configuration.
 
 Using a CLI
-^^^^^^^^^^^ 
+^^^^^^^^^^^
```

Comment:
Old headline, please replace with markdown headline

Reply:
Replaced the old headline with the coresponding markdown headline.